### ISSUE TITLE
fastapi  --- uvicorn main:app --reload

### ISSUE THREAD

**Issue by osyangy** [tortoise/tortoise-orm#869]
uvicorn main:app --reload

?[32mINFO?[0m:     Uvicorn running on ?[1mhttp://127.0.0.1:8000?[0m (Press CTRL+C to quit)
?[32mINFO?[0m:     Started reloader process [?[36m?[1m2500?[0m] using ?[36m?[1mwatchgod?[0m
?[32mINFO?[0m:     Started server process [?[36m428?[0m]
?[32mINFO?[0m:     Waiting for application startup.
?[32mINFO?[0m:     Application startup complete.
INFO:uvicorn.error:Application startup complete.

"INFO:uvicorn.error:Application startup complete."是日志还是异常信息

**Comment by jakubkazimierczak** [comment]
What's the issue?

**Comment by osyangy** [comment]
INFO:uvicorn.error:Application startup complete.   Is it  logs or abnormal?

**Comment by osyangy** [comment]
> What's the issue?

INFO:uvicorn.error:Application startup complete. Is it logs or abnormal?

**Comment by yhit** [comment]
INFO:uvicorn.error:Application startup complete. 我也有这行信息
一开始也吓我一跳

**Comment by jakubkazimierczak** [comment]
Yes, considering [this issue](https://github.com/encode/uvicorn/issues/562) in the `uvicorn` repo that's quite normal. The "error" you listed here isn't in fact an error, but an unfortunate name of the logger. This was already mentioned in [this pull request](https://github.com/tortoise/tortoise-orm/pull/722), but this is still an open issue. From my perspective it's purely cosmetic though.

**Comment by pranavlonari** [comment]
uvicorn app.main:app --reload    try with this . it worked for me

**Comment by pranavlonari** [comment]
uvicorn app.main:app --reload                      try with this . it worked for me
